components/RadioPlayer.vue

```diff
@@ -208,6 +207,16 @@ export default {
       })
       return allStreams
     },
+    arcsiList () {
+      return [...this.$store.state.arcsiShows]
+    },
+    currentShowArcsi () {
+      if (this.np.live.is_live) { // live show
+        return this.arcsiList.find(show => this.slugify(show.name) === this.slugify(this.np.live.streamer_name))
+      } else { // pre-recorded show
+        return this.arcsiList.find(show => this.slugify(show.name) === this.slugify(this.np.now_playing.song.artist))
+      }
+    },
```

Comment:
The basic logic looks good to me. 

Two questions:
- Is it really slugifying? It rather looks to me to be a normalization of special characters so that the strings are comparable. 
- What will this function return if there is no hit?

Reply:
slugifying is because the disrepancies between html charactes in names eg. Erdenklang, or Svadhyaya
it will return null, do you think we need some fallback? but to what?